src/write.c

```diff
@@ -585,6 +586,12 @@ static avifResult avifEncoderAddImageInternal(avifEncoder * encoder,
         if (cellImage->yuvFormat == AVIF_PIXEL_FORMAT_NONE) {
             return AVIF_RESULT_NO_YUV_FORMAT_SELECTED;
         }
+
+        // MIAF (ISO 23000-22:2019), Section 7.3.11.4.1:
+        //   All input image of a grid image item shall use the same chroma sampling format.
```

Comment:
Nit: "image" should be plural "images".

Since this comment is written as a quote, we should quote the exact text from MIAF:

`All input images of a grid image item shall use the same coding format, chroma sampling format, ...`

Reply:
I copy-pasted the line from Draft ISO/IEC 23000-22(E). The typo was likely fixed in a later version.